### DISCUSSION TITLE
Running a script that requires installing packages from a package repository

### DISCUSSION THREAD

**Discussion by fakela** [bacalhau-project/bacalhau#1685] [Q&A] [answered]
How do I run a script that requires installing packages from a package repository like pypi or apt?

**Comment by enricorotundo** [comment, marked as answer]
While networking in <= v0.3.16 was completely disabled, @simonwo has done an amazing job in https://github.com/filecoin-project/bacalhau/pull/1571 to support this use case 🥳 , we expect to release that feature in v0.3.17.

**Reply by enricorotundo** [reply]
Ah caveat, it's only for Docker workloads
